libvncserver/rfbserver.c

```diff
@@ -3412,6 +3507,101 @@ rfbSendNewFBSize(rfbClientPtr cl,
     return TRUE;
 }
 
+/*
+ * Send ExtDesktopSize pseudo-rectangle. This message is used:
+ * - to tell the client to change its framebuffer size
+ * - at the start of the session to inform the client we support size changes through setDesktopSize
```

Comment:
I can't find `rfbSendExtDesktopSize` at session start in the code.

Reply:
Or is it: 
* rfbEncodingExtDesktopSize received in SetEncodings from client
*  cl->useExtDesktopSize = TRUE; 
* rfbSendExtDesktopSize() triggered later
?

Reply:
thanks for the explanation